src/Illuminate/Foundation/Testing/AssertionsTrait.php

```diff
@@ -14,7 +14,7 @@ public function assertResponseOk()
 	{
 		$actual = $this->response->getStatusCode();
 
-		return PHPUnit::assertTrue($this->response->isOk(), 'Expected status code 200, got ' .$actual);
+		return PHPUnit::assertTrue($this->response->isOk(), 'Expected status code 200, got ' . $actual);
```

Comment:
> Our code style is self documented by our code. The same goes for Symfony.

This is not entirely precise for Symfony. We do have [a document explaining our code standard rules](http://symfony.com/doc/current/contributing/code/standards.html).